From 839764542b6149d381f692e7c648240e7f094176 Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <dinguyen@kernel.org>
Date: Mon, 5 Mar 2018 09:21:05 -0600
Subject: [PATCH 22/25] arm64: dts: stratix10: enable watchdog timer on the S10
 devkit

commit  07ecc72ce169d4a4e11259067a714e31d6acd400 from
https://github.com/altera-opensource/linux-socfpga.git

Enables the watchdog0 timer on the Stratix10 devkit.

Signed-off-by: Dinh Nguyen <dinguyen@kernel.org>

Conflicts:
	arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts b/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts
index 81fc603..8f421d6 100644
--- a/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts
+++ b/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts
@@ -173,3 +173,7 @@
 		};
 	};
 };
+
+&watchdog0 {
+	status = "okay";
+};
-- 
2.8.1

